core/resources/lights_frag.glsl

```diff
@@ -0,0 +1,39 @@
+varying vec4 v_amb;
+varying vec4 v_diff;
+varying vec4 v_spec;
+
+vec4 calculateLighting(in vec3 _eyeToPoint, in vec3 _normal) {
+	vec3 eye = vec3(0.0, 0.0, 1.0);
+
+	vec4 amb = v_amb;
+	vec4 diff = v_diff;
+	vec4 spec = v_spec;
+
+//  COMPUTE ALL LIGHTS
+#pragma tangram: lights_to_compute
+
+	//  Final light intensity calculation
+	//
+	vec4 color = vec4(0.0);
+  
+	#ifdef MATERIAL_EMISSION
+	color = g_material.emission;
```

Comment:
Might be better if the function takes a material as input, and can hardcode it to a single global for now (but expand it later if needed?).